src/services/WolfDen.Application/Requests/Queries/Employees/EmployeeDirectory/GetAllEmployeeQueryHandler.cs

```diff
@@ -52,13 +56,13 @@ public async Task<PaginationResponse> Handle(GetAllEmployeeQuery request, Cancel
                    IsActive = e.IsActive
                })
             .ToListAsync(cancellationToken);
-            int totalpage = employees.Count;
-            int pageCount = (int)(Math.Ceiling((decimal)totalpage / request.PageSize));
-            var emp = employees.Skip(pageSize * pageNumber)
-               .Take(pageSize).ToList();
+            /* int totalpage = employees.Count;*/
```

Comment:
Remove commented code if not needed.
Revisit all files to remove unwanted whitespaces and usings
- CTRL + R + CTRL + G : Removes unused usings
- CTRL + K+ CTRL + D : Formats document

Reply:
done